torch_frame/nn/encoder/stype_encoder.py

```diff
@@ -562,16 +570,15 @@ def reset_parameters(self):
 
 
 class LinearEmbeddingEncoder(StypeEncoder):
-    r"""Linear function based encoder for pre-computed embedding features.
-    It applies a linear layer :obj:`torch.nn.Linear(emb_dim, out_channels)`
-    on each embedding feature and concatenates the output embeddings. Note that
-    the implementation does this for all :obj:`text_embedded` features in a
-    batched manner.
+    r"""Linear function based encoder for pre-computed embedding features with
+    the same dimensionalities. It applies a linear layer
+    :obj:`torch.nn.Linear(emb_dim, out_channels)` on each embedding feature and
+    concatenates the output embeddings. Note that the implementation does this
+    for all :obj:`text_embedded` features in a batched manner.
+    NOTE: We will soon shift this to a new LinearMultiEmbeddingEncoder that
+    takes :class:`MultiEmbeddingTensor` object as input.
     """
-
-    # NOTE: We currently support text embeddings but in principle, this encoder
-    # can support any pre-encoded embeddings, including image/audio/graph
-    # embeddings.
+    # TODO: To be deprecated in favaor of LinearMultiEmbeddingEncoder
```

Comment:
Should we deprecate it? Is there any case that user want to use just one linear layer to transform the embeddings (for example user just use one kind of text encoder for all text columns)

Reply:
That's a good point. Personally, I think we can just deprecate this for simplicity. Otherwise, the type of `feat` needs to switch between `Tensor` and `MultiEmbeddingTensor`, which is confusing.

If single kind of text encoder is used, `LinearMultiEmbeddingEncoder` will essentially reduce to `LineariEmbeddingEncoder`.

Reply:
Also, I don't think we will have too many embedding-columns in practice, so for-loop over embedding column should not be too bad.